test/models/user_mailer_test.rb

```diff
@@ -156,7 +156,7 @@ def assert_support_present_in_text(link_text, email = @email)
     setup do
       GovukEnvironment.stubs(:production?).returns(false)
       GovukEnvironment.stubs(:name).returns("test")
-      @the_time = Time.zone.now
+      @the_time = Time.zone.local(2023, 10, 31, 2, 0, 0)
```

Comment:
Subjective: I tend to use `Time.zone.parse("2023-10-31 02:00:00")` in tests, because I think it's more readable.

Reply:
Done, that is nicer, thanks!